### ISSUE TITLE
Dashboard - RTH Stationen bei den RW's rauslösen

### ISSUE THREAD

**Issue by PPPBerlin** [LSS-Manager/lss-manager-v3#257]
Heute ist es mir mit dem GRTW aufgefallen. Bei der RW Anzeige im Dashboard werden bei mir 16 RW's angezeigt, obwohl ich nur 13 + 3 RTH Stationen habe. Das ist also eine verwirrende Anzeige, möglicherweise könnte man dort einfach noch einen 3. Balken einfügen.

**Comment by Suchty112** [comment]
Gute Idee, ich werde mal schauen wie es sich umsetzen lässt. Ist mir zum teil ebend selbst auch passiert, obwohl ich mich dabei aber auch an Sebastians statistik gewendet habe ^^.

**Comment by PPPBerlin** [comment]
Ja, mein HTML/js/css ist etwas eingerostet, sonst würde ich das selber mal machen ;)

**Comment by Suchty112** [comment]
Sry für die etwas vlt späte antwort 😐 Hier wäre jetzt mein Vorschlag wie ich es umsetzen würde. Andersweitig sehe ich aktuell keine Möglichkeit es umzusetzen. @PPPBerlin Wäre das ne Möglichkeit für dich? Da man eh Später mehrere Christophe hat :D . Ist auch erstmal nur explizit für die RTH's vom RD :)
![grafik](https://user-images.githubusercontent.com/1109899/34892532-e40c2598-f7d9-11e7-9c5a-6b32c748fde1.png)
